infrastructure/iam_policy_document.tf

```diff
@@ -0,0 +1,37 @@
+data "aws_iam_policy_document" "iam_pol_doc_for_lambda" {
+  statement {
+    actions = [
+        "s3-object-lambda:Get*",
+        "s3-object-lambda:PutObject",
+        "s3-object-lambda:DeleteObject"
+    ]
+    resources = [
+        "arn:aws:s3:::${local.bucketname-photos}"
+    ]
+  }
+  statement {
+    actions = [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+    ]
+    resources = [
+        "*"
+    ]
+  }
+  statement {
+    actions = [
+        "dynamodb:DescribeStream",
+        "dynamodb:DescribeTable",
+        "dynamodb:Get*",
+        "dynamodb:Query",
+        "dynamodb:Scan",
+        "dynamodb:DeleteItem",
+        "dynamodb:UpdateItem",
+        "dynamodb:PutItem"
+    ]
+    resources = [
+        "arn:aws:dynamodb:us-west-2:${data.aws_caller_identity.current.account_id}:table/kutter-table"
```

Comment:
this is the arn from the resource. 
You can directly refrerence the arn. 

I think like:
`aws_dynamodb_table.dynamodb-kutter-table.arn`

Reply:
I tried it but like this I get an error:
```
╷
│ Error: error updating IAM policy arn:aws:iam::826643085812:policy/nfish-des-pol-lambda_s3_dynamodb: MalformedPolicyDocument: Resource aws_dynamodb_table.dynamodb-kutter-table.arn must be in ARN format or "*".
│       status code: 400, request id: b789e877-cf7b-4787-90a6-afbf38c5ebad
│ 
│   with aws_iam_policy.nfish-des-pol-lambda_s3_dynamodb,
│   on iam_for_lambda.tf line 1, in resource "aws_iam_policy" "nfish-des-pol-lambda_s3_dynamodb":
│    1: resource "aws_iam_policy" "nfish-des-pol-lambda_s3_dynamodb" {
│ 
╵
```

Reply:
now it works